ee/management/commands/sync_replicated_schema.py

```diff
@@ -0,0 +1,65 @@
+import re
+from typing import Dict, Set
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.sql.schema import CREATE_TABLE_QUERIES, build_query, get_table_name
+from posthog.settings import CLICKHOUSE_CLUSTER, CLICKHOUSE_DATABASE, CLICKHOUSE_REPLICATION, MULTI_TENANCY
+
+logger = structlog.get_logger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
Can we add a test for this command? Even just to check that it runs. I'd like to know if I break this accidentally, as it seems a dependency to running in production.

Reply:
Done so! Thanks for pushing me on this - it turns out some tables were sensitive around ordering in `CREATE_TABLE_QUERIES` array that wasn't exposed on my sharded setup (as they were automatically created).